### ISSUE TITLE
Installation from PyPI does not register with cmmnbuild-dep-manager

### ISSUE THREAD

**Issue by levens** [rdemaria/pytimber#40]
Installation from PyPI does not register pytimber with cmmnbuild-dep-manager as the wheel does not include the custom install command. I will look if there is a way around this.

**Comment by rdemaria** [comment]
what if one uses the standard tar.gz instead of the wheel?

**Comment by levens** [comment]
Yes, that seems to work. I have created PR #41 to remove the bdist_wheel option from the release script so that we only upload a sdist to PyPI.
